frontend/src/metabase/query_builder/actions/core/core.ts

```diff
@@ -59,6 +60,22 @@ export { SOFT_RELOAD_CARD };
 export const softReloadCard = createThunkAction(SOFT_RELOAD_CARD, () => {
   return async (dispatch, getState) => {
     const outdatedCard = getCard(getState());
+
+    // It's not entierly clear why this is necessary, but when dispatching
```

Comment:
Idk what I was on when I wrote this comment. I should look into if this still holds true with some of the other changes we've done AND if so try to do a better job of explaining things here.

Reply:
You didn't write this comment, I did... haha

Reply:
The end result of all of this was that we needed to update RTK Query. We can actually delete this `removeQueryResult` stuff. the content verification e2e tests will fail if it causes any issues